exercises/pangram/runtests.jl

```diff
@@ -45,3 +45,7 @@ end
 @testset "pangram with more than 26 letters (if case sensitive)" begin
     @test ispangram("the 1 quick brown fox jumps Over the 2 lazy dogs")
 end
+
+@testset "letters that aren't part of ASCII" begin
+    @test !ispangram("the 1 qüick bröwn föx jümps över the 2 läzy dögs")
```

Comment:
The sentence is obviously nonsense but it demonstrates that a solution using `isletter` would work here because it's a pangram with aou instead of äöü